python/setup.py

```diff
@@ -35,59 +42,143 @@
     cuda_include_dir = os.path.join(os.environ.get('CUDA_HOME'), 'include')
 
 
-rmm_include_dir = '/include'
-rmm_lib_dir = '/lib'
-
-# including cumlprims dirs, in case rmm paths change in the future
-cumlprims_include_dir = '/include'
-cumlprims_lib_dir = '/lib'
+conda_lib_dir = os.path.normpath(sys.prefix) + '/lib'
+conda_include_dir = os.path.normpath(sys.prefix) + '/include'
 
 if os.environ.get('CONDA_PREFIX', None):
     conda_prefix = os.environ.get('CONDA_PREFIX')
-    rmm_include_dir = conda_prefix + rmm_include_dir
-    rmm_lib_dir = conda_prefix + rmm_lib_dir
+    conda_include_dir = conda_prefix + '/include'
+    conda_lib_dir = conda_prefix + '/lib'
+
+
+##############################################################################
+# - Subrepo checking and cloning ---------------------------------------------
+
+
+def clone_repo(name, GIT_REPOSITORY, GIT_TAG, first=False):
+    """
+    Function to clone repos in case they were not cloned by cmake.
+    Variables are named identical to the cmake counterparts for clarity,
+    in spite of not being very pythonic.
+    """
+    if first:
+        subprocess.check_call(['rm', '-rf', 'external'])
```

Comment:
Everything assumes we're in the `build` dir for all of this... I guess that's an ok assumption but worth checking whether it's possible to run from elsewhere? Seeing an `rm -rf` makes me particularly paranoid about that...

Reply:
I'd rather just break out the rm -rf to a `clean_repo` function.. it's surprising to me that clone_repo ever wipes out the directory one level up from the target dir

Reply:
Very good point, will definitely look into a cleaner approach for this